composites/basic/Notification.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { injectIntl, intlShape, defineMessages } from "react-intl";
+
+import Paper from "./Paper";
+import colors from "../../style-guide/colors.json";
+import { Icon } from "../../composites/Plugin/Shared/components/Icon";
+import { times } from "../../style-guide/svg";
+
+const messages = defineMessages( {
+	buttonAriaLabel: {
+		id: "notification.buttonAriaLabel",
+		defaultMessage: "Dismiss this notice",
+	},
+} );
+
+const NotificationContainer = styled.div`
+	display: flex;
+	align-items: center;
+	padding: 20px;
```

Comment:
24px in desktop, 16px in mobile view

Reply:
to fit the 8px grid

Reply:
This change wouldn't match the provided design, see #347. I've measured the padding in the mockup, and it's 20 px, I guess also to match the padding of the notifications in the meta boxes below.
Are we sure we want to change this?

Reply:
@hedgefield

Reply:
The margins in the mockup are an approximation, they haven't been measured out, I'll keep that in mind next time. I would prefer they all adhere to the 24/16 @IreneStr proposed.